tests/resources/OVA-Util.robot

```diff
@@ -0,0 +1,90 @@
+# Copyright 2016-2017 VMware, Inc. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#	http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License
+
+*** Settings ***
+Documentation  This resource provides any keywords related to Unified OVA
+
+*** Variables ***
+${ova_root_pwd}  ova-test-root-pwd
+${ova_appliance_options}  --prop:appliance.root_pwd=${ova_root_pwd} --prop:appliance.permit_root_login=True
+
+${ova_target_vm_name}  vic-unified-ova-integration-test
+${ovftool_options}  --noSSLVerify --acceptAllEulas --name=${ova_target_vm_name} --diskMode=thin --powerOn --X:waitForIp --X:injectOvfEnv --X:enableHiddenProperties
+
+${ova_network_ip0}  10.17.109.207
+${ova_network_netmask0}  255.255.255.0
+${ova_network_gateway}  10.17.109.253
+${ova_network_dns}  10.118.81.1
+${ova_network_searchpath}  eng.vmware.com
+${ova_network_domain}  mrburns
+${ova_network_options}  --prop:network.ip0=${ova_network_ip0} --prop:network.netmask0=${ova_network_netmask0} --prop:network.gateway=${ova_network_gateway} --prop:network.DNS=${ova_network_dns} --prop:network.searchpath=${ova_network_searchpath} --prop:network.domain=${ova_network_domain}
+
+${ova_harbor_admin_password}  harbor-admin-passwd
+${ova_harbor_db_password}  harbor-db-passwd
+${ova_service_options}  --prop:harbor.admin_password=${ova_harbor_admin_password} --prop:harbor.db_password=${ova_harbor_db_password}
+
+${ova_options}  ${ovftool_options} ${ova_appliance_options} ${ova_service_options}
+${ova_options_with_network}  ${ova_options} ${ova_network_options} 
+
+${tls_not_disabled}  False
+
+# Requires vc credential for govc
```

Comment:
not sure what this comment is for.. from the looks of your other comment on the 2nd keyword, maybe move this comment below the *** Keywords *** line